modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/metric/IoStatisticsHolder.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.ignite.internal.pagememory.metric;
+
+/**
+ * Holder of IO statistics.
+ */
+public interface IoStatisticsHolder {
+    /**
+     * Track logical read of given page.
+     *
+     * @param pageAddr Address of page.
+     */
+    public void trackLogicalRead(long pageAddr);
```

Comment:
What does 'track' mean here? Increment the corresponding counter? If so, then (although I'm not an expert in the terminology) 'track' seems a bit vague. 'register'? Or simply 'on', like 'onLogicalRead()`?

On the other hand, if we name it `track`, then probably it makes sense to call the type 'Tracker', not 'Holder'?

Just thoughts... naming is hard.

Reply:
I agree, it's hard, I didn't consider renaming anything if it was possible